pkg/tsdb/postgres/postgres_test.go

```diff
@@ -254,7 +254,7 @@ func TestPostgres(t *testing.T) {
 			require.Len(t, frames, 1)
 			require.Len(t, frames[0].Fields, 17)
 
-			require.Equal(t, int16(1), *frames[0].Fields[0].At(0).(*int16))
+			require.Equal(t, int16(1), frames[0].Fields[0].At(0).(int16))
```

Comment:
For this one, this column is not nullable, so the int16s do not end up as pointers anymore.